ReactiveCocoa/Swift/Property.swift

```diff
@@ -104,22 +104,15 @@ public final class MutableProperty<Value>: MutablePropertyType {
 	/// Need a recursive lock around `value` to allow recursive access to
```

Comment:
This comment is no longer relevant here. Can you move it to where you're instantiating the particular lock in the constructor?